src/content/docs/guides/create/index.mdx

```diff
@@ -65,6 +64,77 @@ You'll now see a new window open with your app running.
 
 **Congratulations!** You've made your Tauri app! 🚀
 
+## Manual Setup using the Tauri CLI
+
+To get started manually creating a new Tauri project using the Tauri CLI, follow the steps below:
+
+<Steps>
+
+    1. First install Tauri's CLI tool globally using your package manager of choice:
+
+        <CommandTabs
+            npm="npm install -g @tauri-apps/cli"
+            yarn="yarn global add @tauri-apps/cli"
+            pnpm="pnpm add -g @tauri-apps/cli"
+            cargo="cargo install tauri-cli"
+        />
+
+    2. Create a new directory for your project and initialize a your app. You can use any frontend framework you prefer, like NextJS, Nuxt, Svelte, or just plaing HTML, CSS, and JavaScript.
+
+        <CommandTabs
+            npm="mkdir my-tauri-app
+                cd my-tauri-app
+                npm <command to bootstrap a template for your frontend app>"
+            yarn="mkdir my-tauri-app
+                cd my-tauri-app
+                yarn <command to bootstrap a template for your frontend app>"
+            pnpm="mkdir my-tauri-app
+                cd my-tauri-app
+                pnpm <command to bootstrap a template for your frontend app>"
+        />
```

Comment:
I think that instead we can focus on the end goal that is to have a parent folder 'my-tauri-app' and a folder for the frontend app that can be called anything else 'my-tauri-frontend'. This can be a use case for the starlight [file tree](https://starlight.astro.build/guides/components/#file-tree).

Not exactly this wording, just a thought
```suggestion
      In this directory setup the frontend of your choice ...
      
      The folder structure should be
      - my-tauri-app
               - my-frontend
                           ....frontendfiles
```

what do you think?

Reply:
I think that we should just give the user the choice to manage their own project however they want. It could be creating a directory or just live in the root tho. I don't see no problem with this part.

Reply:
then perhaps I think it's needed to add the cargo command for the front end aswell

Reply:
Will cargo work on this part?

Reply:
I haven't used, but I believe there's no need to use node based frontend
for example:
https://trunkrs.dev/ cargo install --locked trunk

And so does Sycamore, each will have different usage but are cargo based

Reply:
https://tauri.by.simon.hyll.nu/samples/simple/

Reply:
gotcha, makes sense.